GDJS/Runtime/pixi-renderers/pixi-filters-tools.ts

```diff
@@ -14,7 +14,7 @@ namespace gdjs {
       return arr.indexOf(value) !== -1 ? value : min;
     };
 
-    const _filterCreators: {
+    export const _filterCreators: {
```

Comment:
Seems weird to export something that has an underscore? Either we need to rename this, or use something else.

Reply:
I think I checked it out by mistake (I was trying to figure out what are these objects exactly meant for). My bad, I'll fix it. 😬